### PR TITLE
[GH#52915] clarified single-az cluster minimum infra nodes

### PR DESCRIPTION
**Pull request by pfeifferj**
Version(s):
ROSA

Issue:
[#52915](https://github.com/openshift/openshift-docs/issues/52915)

Link to docs preview:

[About availability for Red Hat OpenShift Service on AWS](https://56277--docspreview.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-policy-understand-availability.html)

QE review:
- [ ] QE has approved this change.

Additional information:

[Source](https://catalog.workshops.aws/aws-openshift-workshop/en-US/2-networking/2-rosa-az) for Single-AZ Cluster setup.

### CODE REVIEW

**Comment by bmcelvee** [comment]
Thank you for opening this PR, @pfeifferj! I'm going to leave some requested edits in line, and request the additional reviews required to merge. :)

File: modules/rosa-policy-failure-points.adoc [hunk 1]
```diff
@@ -35,7 +35,9 @@ When accounting for possible node failures, it is also important to understand h
 
 [id="rosa-policy-container-cluster-failure_{context}"]
 == Cluster failure
-ROSA clusters have at least three control plane nodes and three infrastructure nodes that are preconfigured for high availability, either in a single zone or across multiple zones, depending on the type of cluster you have selected. Control plane and infrastructure nodes have the same resiliency as worker nodes, with the added benefit of being managed completely by Red Hat.
+Single-AZ ROSA clusters have at least three control plane and two infrastructure nodes placed into the smae AZ in the private subnet.
```
Comment (bmcelvee) [inline modules/rosa-policy-failure-points.adoc:38]: s/placed into the smae AZ/in the same availability zone (AZ)

**Comment by bmcelvee** [comment]
@arendej Would you mind giving this PR a review, please? Thanks!

**Review by arendej — APPROVED** [review]
thank you.
approved.

**Comment by bmcelvee** [comment]
No QE review required.

**Pull request merged**